ui/lib/replication/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "replication",
+  "keywords": [
+    "ember-addon",
+    "ember-engine"
+  ],
+  "dependencies": {
```

Comment:
ah okay, so we need to add all engine-specific dependencies to each engine's `package.json` as well as the `core/package.json`?

Reply:
Oh nice catch! We might be able to get away with removing these two because we rely on `core` which relies on them. They were part of the default blueprint when generating an in-repo-engine and you'd normally need them out of the box for template and cross-compilation concerns. I'll try removing them and see if that works.

Reply:
Aha so it looks like these are required because of template compilation. Removing them I got these errors and a failed build:

```
DEPRECATION: Addon files were detected in `/Users/meirish/go/src/github.com/hashicorp/vault/ui/lib/replication/addon`, but no JavaScript preprocessors were found for `replication`. Please make sure to add a preprocessor (most likely `ember-cli-babel`) to in `dependencies` (NOT `devDependencies`) in `replication`'s `package.json`.
Addon templates were detected, but there are no template compilers registered for `replication`. Please make sure your template precompiler (commonly `ember-cli-htmlbars`) is listed in `dependencies` (NOT `devDependencies`) in `replication`'s `package.json`.
```